src/it/scala/io/iohk/ethereum/sync/util/FakePeerFastSync.scala

```diff
@@ -0,0 +1,77 @@
+package io.iohk.ethereum.sync.util
+
+import akka.actor.ActorSystem
+import cats.effect.Resource
+import io.iohk.ethereum.Mocks.MockValidatorsAlwaysSucceed
+import io.iohk.ethereum.blockchain.sync.FastSync
+import io.iohk.ethereum.mpt.{HashNode, MptNode, MptTraversals}
+import io.iohk.ethereum.sync.util.SyncUtils.retryUntilWithDelay
+import monix.eval.Task
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+class FakePeerFastSync(peerName: String) extends FakePeer {
+
+  override implicit protected def system: ActorSystem = ActorSystem(peerName)
```

Comment:
Is it possible to have 1 actor system per test suite (test class) instead of one per test? Maybe pass actor system as param. WDYT

BTW Since this is def so ... it looks like we create ActorSystem every time we call `system`.

Reply:
I understood the approach. Only that for every test you need to build the actor system and build the tree with new sync.
Anyways. I removed this class.